### DISCUSSION TITLE
calculating swell height in ww3 v6.07

### DISCUSSION THREAD

**Discussion by JMSnyder1980** [NOAA-EMC/WW3#1023] [General]
How do I calculate swell height or NOAA parameter 105 from the ww3 v6.07 output?   From this output I have the following parameters hs,fp,dp, and wnd.   How do I calculate the swell height from these?   Or is there a fixed swell height parameter in the model data itself?

**Comment by Liu-Jincan** [comment]
Add the 'PHS' variable in the `ww3_shel.nml` or `ww3_multi.nml`.
